src/containers/PlanDetails/PlanDetails.tsx

```diff
@@ -118,6 +118,7 @@ const PlanDetails = ({
   return (
     <WrapStyled>
       <GracePeriodError />
+      {/* TODO: implement headers hierachy (dynamic calculation) */}
```

Comment:
if we have it in JIRA ticket - we can remove this comment

Reply:
done here as well